lib/kino_db/connection_cell.ex

```diff
@@ -23,7 +23,7 @@ defmodule KinoDB.ConnectionCell do
       "hostname" => attrs["hostname"] || "localhost",
       "database_path" => attrs["database_path"] || "",
       "port" => attrs["port"] || default_port,
-      "use_ipv6" => false,
+      "use_ipv6" => attrs["use_ipv6"] || false,
```

Comment:
```suggestion
      "use_ipv6" => Map.get(attrs, "use_ipv6", false),
```

Technically `false || false` works as expected, but for booleans it's always a good idea to be explicit in this regard :)